src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/security/CcdEventAuthorizor.java

```diff
@@ -24,6 +29,8 @@ public CcdEventAuthorizor(
     public void throwIfNotAuthorized(
         Event event
     ) {
+        long startTime = System.currentTimeMillis();
```

Comment:
This is fine for what we want here.  In future we could consider something like:  https://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/time/StopWatch.html

Don't think we need it here though.

Reply:
I am thinking of generic IA http client (RestTemplate wrapper) for 3rd party services with proper response logging (in case of exception) and optional timing (env flag), it would be more elegant. What do you think?

Reply:
I think some generic standardised logging for our interactions with other services would be good.  I think I prefer it being separate from the RestTemplate however.  Maybe we can re-use what Spring gives us with @ControllerAdvice and add some generic logging there?